src/WebJobs.Script/Description/Node/Script/functionTemplate.js

```diff
@@ -40,4 +42,36 @@ return function (context, callback) {{
     delete context.inputs;
 
     f.apply(null, inputs);
-}};
\ No newline at end of file
+}};
+
+function getEntryPoint(f, context) {{
```

Comment:
As an interesting meta point, I'd eventually like to introduce some Node Mocha tests for this framework code. To do so, I'll hoist all this fx logic out into its own module, exporting multiple testable functions (which was the point of this PR for users :))